google-cloud-talent/lib/google/cloud/talent/v4beta1/batch_pb.rb

```diff
@@ -4,6 +4,7 @@
 
 require 'google/protobuf'
 
+require 'google/api/annotations_pb'
```

Comment:
This requires an updated dependency on googleapis-common-protos version 1.3.9 or higher.